resources/status.json

```diff
@@ -13,6 +13,9 @@
     },
     "hard": {
       "level": 0
+    },
```

Comment:
Forget to add a secret level. Themes and status MUST be synchronized. When we'll build status from code this problem should disappear.